AsyncDisplayKit/ASPagerFlowLayout.m

```diff
@@ -54,6 +56,11 @@ - (NSIndexPath *)_indexPathForVisiblyCenteredItem
   return nil;
 }
 
+- (BOOL)_dataSourceIsEmpty
+{
+  return ([self.collectionView numberOfSections] > 0 && [self.collectionView numberOfItemsInSection:0] > 0);
```

Comment:
Why is data source empty if the collection view has at least 1 section and 1 row?